tomb/src/types/config/bucket/omni.rs

```diff
@@ -204,8 +195,9 @@ impl OmniBucket {
         global: &mut GlobalConfig,
         client: &mut Client,
     ) -> Result<String, TombError> {
-        let local_deletion = if let Some(local) = &self.local &&
-        prompt_for_bool("are you sure you want to delete this Bucket locally?") {
+        let local_deletion = if let Some(local) = &self.local
+            && prompt_for_bool("are you sure you want to delete this Bucket locally?")
+        {
```

Comment:
And here (looks like there are a bunch more, I'm going to stop commenting on the individual instances)